### ISSUE TITLE
Android 16

### ISSUE THREAD

**Issue by Pinned4** [Nitsuya/DoNotTryAccessibility#10]
I tried to compile the source and error out on a debug line. It cannot hide accessibility on Android 16.
Please update!
